### PR TITLE
Implement `dedi_sendPrintsToClient`

### PR DESCRIPTION
**Pull request by BobTheBob9**
Implements the convar `dedi_sendPrintsToClient` that allows the dedicated server to forward all logs to connected clients

Requires
- [x] #411

### CODE REVIEW

**Comment by BobTheBob9** [comment]
oh this diff fucking sucks beacuse the pr is based on the g_pGlobals pr cool

**Comment by GeckoEidechse** [comment]
Out of curiosity and so that I'm up to date, what's the use case for this? Debugging help, server management, sth else? ^^

**Comment by GeckoEidechse** [comment]
There seem to be quite some unrelated changes in there. Do we maybe wanna splice them out a bit to get a cleaner diff? ^^

**Comment by BobTheBob9** [comment]
> There seem to be quite some unrelated changes in there. Do we maybe wanna splice them out a bit to get a cleaner diff? ^^

seems to be due to the branch it was based on (pr that is now merged) being included in changes

**Comment by GeckoEidechse** [comment]
> > There seem to be quite some unrelated changes in there. Do we maybe wanna splice them out a bit to get a cleaner diff? ^^
> 
> seems to be due to the branch it was based on (pr that is now merged) being included in changes

That should be exactly not the case since the base branch was merged (unless I'm going insane). Unless you're faster, imma splice out some fix to the extent that I understand to reduce the diff a bit ^^

File: NorthstarDLL/engine/r2engine.h [hunk 1]
```diff
@@ -201,4 +201,64 @@ namespace R2
 	extern server_state_t* g_pServerState;
 
 	extern char* g_pModName;
+
+	// clang-format off
+	OFFSET_STRUCT(CGlobalVars)
```
Comment (BobTheBob9) [inline NorthstarDLL/engine/r2engine.h:206]: this whole struct is the main thing cglobals-pr adds, so i think it's safe to say something has gone wrong given its in the diff for this pr

Comment (GeckoEidechse) [reply NorthstarDLL/engine/r2engine.h]: Merging main back into this branch resolved diff issues.

File: NorthstarDLL/NorthstarDLL.vcxproj [hunk 2]
```diff
@@ -549,4 +551,4 @@
   <ImportGroup Label="ExtensionTargets">
     <Import Project="$(VCTargetsPath)\BuildCustomizations\masm.targets" />
   </ImportGroup>
-</Project>
+</Project>
```
Comment (GeckoEidechse) [inline NorthstarDLL/NorthstarDLL.vcxproj:554]: You stripped newline, reeee >:(

File: NorthstarDLL/NorthstarDLL.vcxproj.filters [hunk 3]
```diff
@@ -1429,10 +1432,13 @@
...
+    </ClCompile>
   </ItemGroup>
   <ItemGroup>
     <MASM Include="audio_asm.asm">
-      <Filter>Source Files</Filter>
+      <Filter>Source Files\client</Filter>
     </MASM>
   </ItemGroup>
-</Project>
+</Project>
```
Comment (GeckoEidechse) [inline NorthstarDLL/NorthstarDLL.vcxproj.filters:1444]: You stripped newline, reeee >:(

**Review by GeckoEidechse — APPROVED** [review]
With `dedi_sendPrintsToClient` set `1` dedi sends prints to client as expected.

**Comment by GeckoEidechse** [comment]
Not sure if we need a test for docker additionally.

File: NorthstarDLL/NorthstarDLL.vcxproj (hunk 2 above)
Comment (GeckoEidechse) [reply NorthstarDLL/NorthstarDLL.vcxproj:554]: Seems like visual studio did it, unfortunate but w/e

File: NorthstarDLL/NorthstarDLL.vcxproj.filters (hunk 3 above)
Comment (GeckoEidechse) [reply NorthstarDLL/NorthstarDLL.vcxproj.filters:1444]: Seems like visual studio did it, unfortunate but w/e

**Pull request merged**